src/ansi-c/scanner.l

```diff
@@ -473,6 +473,13 @@ void ansi_c_scanner_init()
                     return make_identifier();
                 }
 
+"_Float128"     { // clang doesn't have it
```

Comment:
You could reduce the code duplication by adding that to the `"__float128"` block as `"__float128" | "_Float128"` as is done for, e.g., `"__complex__" | "__complex"`.